bakerydemo/settings/base.py

```diff
@@ -226,30 +227,14 @@
 
 # Only enable CSP when enabled through environment variables.
 if "CSP_DEFAULT_SRC" in os.environ:
-    MIDDLEWARE.append("csp.middleware.CSPMiddleware")
-
-    # Only report violations, don't enforce policy
-    CSP_REPORT_ONLY = True
-
-    # The “special” source values of 'self', 'unsafe-inline', 'unsafe-eval', and 'none' must be quoted!
-    # e.g.: CSP_DEFAULT_SRC = "'self'" Without quotes they will not work as intended.
-
-    CSP_DEFAULT_SRC = os.environ.get("CSP_DEFAULT_SRC").split(",")
-    if "CSP_SCRIPT_SRC" in os.environ:
-        CSP_SCRIPT_SRC = os.environ.get("CSP_SCRIPT_SRC").split(",")
-    if "CSP_STYLE_SRC" in os.environ:
-        CSP_STYLE_SRC = os.environ.get("CSP_STYLE_SRC").split(",")
-    if "CSP_IMG_SRC" in os.environ:
-        CSP_IMG_SRC = os.environ.get("CSP_IMG_SRC").split(",")
-    if "CSP_CONNECT_SRC" in os.environ:
-        CSP_CONNECT_SRC = os.environ.get("CSP_CONNECT_SRC").split(",")
-    if "CSP_FONT_SRC" in os.environ:
-        CSP_FONT_SRC = os.environ.get("CSP_FONT_SRC").split(",")
-    if "CSP_BASE_URI" in os.environ:
-        CSP_BASE_URI = os.environ.get("CSP_BASE_URI").split(",")
-    if "CSP_OBJECT_SRC" in os.environ:
-        CSP_OBJECT_SRC = os.environ.get("CSP_OBJECT_SRC").split(",")
-    if "CSP_FRAME_SRC" in os.environ:
-        CSP_FRAME_SRC = os.environ.get("CSP_FRAME_SRC").split(",")
-    if "CSP_REPORT_URI" in os.environ:
-        CSP_REPORT_URI = os.environ.get("CSP_REPORT_URI")
+    MIDDLEWARE.append("django.middleware.csp.ContentSecurityPolicyMiddleware")
+    # Gravatar images should be disabled for strict CSP
+    WAGTAIL_GRAVATAR_PROVIDER_URL = None
+
+# To enforce a CSP policy in report-only mode:
+SECURE_CSP_REPORT_ONLY = {
+    "default-src": [CSP.SELF, "*.wagtail.org"],
+    # Add more directives as needed.
+}
```

Comment:
Could you update this so we can preserve how this is done with environment variables? I think ideally we would want to preserve our ability to configure this with environment variables, just do this with the vanilla Django implementation.

I think this should also be done only when `if "CSP_DEFAULT_SRC" in os.environ:`.